core/src/main/java/com/amplifyframework/core/model/ModelSchema.java

```diff
@@ -396,7 +423,7 @@ public int hashCode() {
         return ObjectsCompat.hash(
                 getName(),
                 getPluralName(),
-                hasOwnerAuthorization(),
+                authRules(),
```

Comment:
Rename as `getAuthRules()`, the other ones use the `get<Foo>` prefix.

Reply:
Done!